pnet_test.go

```diff
@@ -3,31 +3,31 @@ package ipfscluster
 import "testing"
 
 func TestClusterSecretFormat(t *testing.T) {
-    goodSecret := "0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
-    emptySecret := ""
-    tooShort := "0123456789abcdef"
-    tooLong := "0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef0"
-    unsupportedChars := "0123456789abcdef0123456789!!!!!!0123456789abcdef0123456789abcdef"
+	goodSecret := "0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
```

Comment:
Yeah, this was the result of me running `go fmt` (which I'd forgotten to do before). Easy enough to change, I'll make another PR with this fix.